app/assets/stylesheets/details.scss

```diff
@@ -12,3 +12,8 @@
 td.file-level {
   padding-left: var(--depth);
 }
+
+/* zebra-stripe files table on edit page */
+tr.attached-files:nth-child(even) {
+  background-color: #dadada;
```

Comment:
[![Watch the video](https://img.youtube.com/vi/LnBSc28Obs4/default.jpg)](https://youtu.be/LnBSc28Obs4)

Reply:
is this 😎 or WTF? or both

Reply:
Oh, it's a link to the music video for the song, "Da Da Da"